report/report-server/src/main/kotlin/com/testerum/report_server/service/ReportService.kt

```diff
@@ -0,0 +1,42 @@
+package com.testerum.report_server.service
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import com.testerum.report_generators.reports.report_model.template.ManagedReportsExecutionListener
+import com.testerum.report_generators.reports.utils.EXECUTION_LISTENERS_OBJECT_MAPPER
+import com.testerum.runner.events.model.ConfigurationEvent
+import com.testerum.runner.events.model.RunnerEvent
+import org.springframework.stereotype.Service
+import java.nio.file.Path
```

Comment:
Use ``JavaPath``, like in the rest of the project:
```suggestion
import java.nio.file.Path as JavaPath
```